src/latest_release_note.ts

```diff
@@ -1,26 +1,39 @@
+</div>
+
+<h2>📝(April 21, 2020)</h2>
```

Comment:
_&ast;nit&ast;_
Perhaps omit the brackets around the date?

Reply:
Updated!